pkg/pillar/utils/cloudconfig/cloudconfig.go

```diff
@@ -105,3 +130,27 @@ func WriteFile(log *base.LogObject, file WritableFile, rootPath string) error {
 
 	return nil
 }
+
+func decodeGzipContent(content string) ([]byte, error) {
+	gzipReader, err := gzip.NewReader(strings.NewReader(content))
+	if err != nil {
+		return nil, err
+	}
+	defer gzipReader.Close()
+	contentBytes, err := io.ReadAll(io.LimitReader(gzipReader, MaxDecompressedContentSize))
```

Comment:
```suggestion
	contentBytes, err := io.ReadAll(io.LimitReader(gzipReader, MaxDecompressedContentSize+1))
```
I understand the comment https://github.com/lf-edge/eve/pull/3553/files#diff-a5680bb7f3f3a3212b4c6f6f870994611ac0c00d74d6c5f7a1037e4cb5a5e426R38 that `MaxDecompressedContentSize` should still be allowed.

Reply:
Makes sense. Fixed!